app/pages/lab-pages-editor/components/TasksPage.jsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-console */
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable react/require-default-props */
+/* eslint-disable radix */
+
+import { useWorkflowContext } from '../context.js';
+import strings from '../strings.json'; // TODO: move all text into strings
+
+import GripIcon from '../icons/GripIcon.jsx';
+
+export default function TasksPage() {
+  const { workflow, update } = useWorkflowContext();
+  const isActive = true; // TODO
+
+  function placeholderEventHandler() {
+    console.log('+++ TODO');
+  }
+
+  console.log('+++ workflow: ', workflow);
+
+  if (!workflow) return null;
+
+  return (
+    <div className="tasks-page">
+      <div className="workflow-title flex-row">
+        <h2 className="flex-item">{workflow?.display_name}</h2>
+        <span className="workflow-id">{`#${workflow?.id}`}</span>
+        {(isActive) ? <span className="status-active">Active</span> : <span className="status-inactive">Inactive</span>}
+      </div>
+      <section>
+        <h3>Tasks</h3>
+        <div className="flex-row">
+          <button
+            className="flex-item big primary"
+            onClick={placeholderEventHandler}
+            type="button"
+          >
+            Add a new Task +
+          </button>
+          {/* Dev observation: the <select> should have some label to indicate it's for choosing the starting task. */}
+          <select
+            aria-label="Choose starting test"
+            className="flex-item"
+          >
+            <option disabled={true}>Choose starting test</option>
+          </select>
+        </div>
+        <ul className="steps-list">
```

Comment:
All HTML landmark elements can be labelled, so you can label this list with either `aria-label` or `aria-labelledby`. 

Here's an example of a list with a linked heading, from project Team pages. 

https://github.com/zooniverse/front-end-monorepo/blob/76d0d94d2aa348b3a5ded97b76cff050ef67d3c3/packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPage.js#L100-L119

Reply:
[HTML landmarks](https://developer.mozilla.org/en-US/blog/aria-accessibility-html-landmark-roles/) can also be labelled, which can be useful for navigating complex pages, like these project builder pages. 

Use ctrl-opt-U to open the rotor in VoiceOver, then navigate to Landmarks to see a navigable list of page landmarks.

Reply:
Noted; (1) I'll read up on the MDN article, and (2) add labels to the lists.

Thanks again - I wasn't aware that lists could/should be labelled.

Reply:
Depends how complex the page is, to be honest. Sometimes, adding extra labels to lists can make them more confusing. 

Here, labelling the list as ‘edit tasks’ or ‘order tasks’ might help explain the purpose of the list.